src/core/sock.c

```diff
@@ -438,6 +456,16 @@ int nn_sock_getopt_inner (struct nn_sock *self, int level,
             strncpy (optval, self->socket_name, *optvallen);
             *optvallen = strlen(self->socket_name);
             return 0;
+        case NN_SEC_ATTR:
+            memcpy (optval, self->sec_attr, self->sec_attr_size);
```

Comment:
For reference, here's the nanomsg style guide for consistency: http://nanomsg.org/development-style.html

Style guide requires space before the open paren.